src/client/Microsoft.Identity.Client/Instance/Region/RegionManager.cs

```diff
@@ -46,6 +46,11 @@ public RegionInfo(string region, RegionAutodetectionSource regionSource, string
         private static bool s_failedAutoDiscovery = false;
         private static string s_regionDiscoveryDetails;
 
+        private readonly LinearRetryPolicy _linearRetryPolicy = new LinearRetryPolicy(
```

Comment:
Instead of repeating all this ctor code everywhere, can you have a static method like `LinearRetryPolicy.CreateStsRetry()` . This way you can hide those constants.

Reply:
Or have a new object `RetryPolicyFactory.CreateStsRetry()`

Reply:
I'll make a separate PR for a RetryPolicyFactory since this PR is so big already.